### PR TITLE
fix to handle under maintenance status

### PR DESCRIPTION
**Pull request by basotia**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-CHANGE-ME](ADD URL HERE)

##### Before Reviewing
- [ ]  Have you pulled the branch down?
- [ ]  Have you assigned yourself to the PR?
- [ ]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?
- [ ]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests? (where applicable)
- [ ]  Have you addressed warnings where appropriate?
- [ ]  Have you rebased against the current version of main?
- [ ]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -336,4 +336,5 @@ taskList.agent.closeCompanyLoansWrittenOffReleasedTask = Benthyciadau gan gwmnï
 notStarted = Heb ddechrau hyd yn hyn
 completed = Wedi cwblhau
 checkNow = Gwirio
-inProgress = Ar y gweill
\ No newline at end of file
+inProgress = Ar y gweill
+underMaintenance = Wrthi’n cynnal a chadw’r safle
```
Comment (matthewwardcap) [inline conf/messages.cy:340]: Duplicate of:

`common.underMaintenance = Wrthi’n cynnal a chadw’r safle`

https://github.com/hmrc/income-tax-submission-frontend/blob/e0fae9dfc06aced834b7abba53c0a201ebfd144a/conf/messages.cy#L21

File: conf/messages [hunk 2]
```diff
@@ -349,3 +349,4 @@ notStarted = Not yet started
 completed = Completed
 checkNow = Check
 inProgress = In progress
+underMaintenance = Under Maintenance
```
Comment (matthewwardcap) [inline conf/messages:352]: Duplicate of:

`common.underMaintenance = Under maintenance`

https://github.com/hmrc/income-tax-submission-frontend/blob/e0fae9dfc06aced834b7abba53c0a201ebfd144a/conf/messages#L21

**Review by matthewwardcap — CHANGES_REQUESTED** [review]
Few duplicate messages that are causing tests to fail

File: conf/messages.cy (hunk 1 above)
Comment (basotia) [reply conf/messages.cy:339]: done

File: conf/messages (hunk 2 above)
Comment (basotia) [reply conf/messages]: done

**Review by matthewwardcap — APPROVED** [review]

**Pull request merged**